setup.py

```diff
@@ -106,7 +106,7 @@ def checkProgramIsInstalled(self, program, args, where_to_download,
 setup(
     name='pyGenomeTracks',
     version=get_version(),
-    author='Fidel Ramírez, Vivek Bhardwaj, Joachim Wolf, Björn Grüning',
+    author='Lucille Lopez-Delisle, Leily Rabbani, Joachim Wolf, Björn Grüning',
     author_email='deeptools@googlegroups.com',
```

Comment:
is anyone taking care of this google group? If not we should remove the reference to it in all documents.

Reply:
Perfect I am not part of it...

Reply:
What should I put instead? Can Freiburg uni provide us a mailing list? (For info it is also on hicexplorer)

Reply:
We simply don't provide a mail?

Reply:
ok done